File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.java
import java.util.*;

public class Main {
    public static void main(String[] args) {  
        try (Scanner sc = new Scanner(System.in)) {
            int T = sc.nextInt(); 
            for(int i=0;i<T;i++){
                int h = sc.nextInt();
                int w = sc.nextInt();
                int n = sc.nextInt();

                int room=0;
                int floor=0;
                
                if(n%h==0){
                    room = n/h;
                    floor = h;
                }else{
                    room = n/h +1;
                    floor = n%h;
                }
                System.out.println(floor*100+room);
            }
        }
    }
}

File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.py
T = int(input())

for i in range(T):
    h,w,n = map(int,input().split())
    
    if(n>h):
        if(n%h==0):
            floor = h
            room = n//h
        else:    
            room = n//h + 1
            floor = n%h
    else:
        room = 1
        floor = n    
    print(floor*100+room)